features/tags_and_wrangling/tag_wrangling_more.feature

```diff
@@ -104,3 +104,105 @@ Feature: Tag wrangling: assigning wranglers, using the filters on the Wranglers
     Then "Testing" should not be assigned to the wrangler "tangler"
     When I edit the tag "Testing"
     Then I should see "Sign Up"
+
+  Scenario: Updating multiple tags works.
+    Given the following typed tags exists
+        | name                                   | type         |
+        | Cowboy Bebop                           | Fandom       |
+        | Spike Spiegel is a sweetie             | Freeform     |
+        | Jet Black is a sweetie                 | Freeform     |
+      And I am logged in as a random user
+      And I post the work "Brain Scratch" with fandom "Cowboy Bebop" with freeform "Spike Spiegel is a sweetie"
+      And I post the work "Asteroid Blues" with fandom "Cowboy Bebop" with freeform "Jet Black is a sweetie"
+     When the tag wrangler "lain" with password "lainnial" is wrangler of "Cowboy Bebop"
+       And I follow "Tag Wrangling"
+       And I follow "2"
+       And I fill in "fandom_string" with "Cowboy Bebop"
+       And I check the mass wrangling option for "Spike Spiegel is a sweetie"
+       And I check the mass wrangling option for "Jet Black is a sweetie"
+       And I press "Wrangle"
+     Then I should see "The following tags were successfully wrangled to Cowboy Bebop: Spike Spiegel is a sweetie, Jet Black is a sweetie"
+
+  Scenario: Updating multiple tags works and set them as canonical
+    Given the following typed tags exists
+        | name                                   | type         | canonical |
+        | Cowboy Bebop                           | Fandom       | true      |
+        | Faye Valentine is a sweetie            | Freeform     | false     |
+        | Ed is a sweetie                        | Freeform     | false     |
+      And I am logged in as a random user
+      And I post the work "Asteroid Blues" with fandom "Cowboy Bebop" with freeform "Ed is a sweetie"
+      And I post the work "Honky Tonk Women" with fandom "Cowboy Bebop" with freeform "Faye Valentine  is a sweetie"
```

Comment:
It doesn't affect anything because they'd be removed when posting the work, but there's an extra space between `Valentine` and `is a sweetie` here.